code/src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import GitIcon from '../assets/github.png';
```

Comment:
Another way of solving this if you would like more social media icons is to import Font Awesome icons (like we did but we put it in the footer). We wanted to post code but for some reason it does not display in the comment?

To make the icons show one also has to import them in the top of the component. If you would like to know more then we can discuss it on the Monday session!